app/views/shared/_modal.html.slim

```diff
@@ -0,0 +1,20 @@
+- root_attributes ||= {}
+- root_attributes[:id] ||= id
+- title ||= ''
+- size ||= ''
+- header = true if header.nil?
+- footer_button_text ||= ''
+
+.modal.fade *root_attributes aria-hidden="true" tabindex="-1"
+  .modal-dialog.modal-dialog-centered class=size
+    .modal-content
+      - if header
+        .modal-header
+          h6.modal-title = title
+          button.btn-close data-bs-dismiss="modal" aria-label="Close"
+
+      = yield
+
+      - if footer_button_text.blank?
+        .modal-footer
+          button.btn.btn-light data-bs-dismiss="modal" = footer_button_text
```

Comment:
I would tend to keep this more generic than just displaying a button to dismiss the modal (i.e., with the option to move arbitrary tags to the footer).

Reply:
As I wrote above I am not sure how to do this (how to use multiple blocks?).